modules/wine-deps.yml

```diff
@@ -0,0 +1,136 @@
+    sources:
+      - type: git
+        url: "https://github.com/KhronosGroup/OpenCL-Headers.git"
+        commit: c5a4bbeabb10d8ed3d1c651b93aa31737bc473dd
+
+  - name: libpcap
+    config-opts:
+      - --enable-ipv6
+    sources:
+      - type: archive
+        url: "https://www.tcpdump.org/release/libpcap-1.9.0.tar.gz"
+        sha256: 2edb88808e5913fdaa8e9c1fcaf272e19b2485338742b5074b9fe44d68f37019
+    cleanup:
+      - /bin
+
+  - name: v4l2-utils
+    config-opts:
+      - --disable-v4l-utils
+      - --disable-qv4l2
+      - --disable-qvidcap
+    sources:
+      - type: archive
+        url: "https://linuxtv.org/downloads/v4l-utils/v4l-utils-1.20.0.tar.bz2"
+        sha256: 956118713f7ccb405c55c7088a6a2490c32d54300dd9a30d8d5008c28d3726f7
+
+  - name: libgphoto2
+    sources:
+      - type: archive
+        url: "https://github.com/gphoto/libgphoto2/archive/libgphoto2-2_5_23-release.tar.gz"
+        sha256: 8de52fd997aceda895abad5d8d95a888bce24a1c739079cff64dae1da7039dde
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -ifv
+    cleanup:
+      - /bin
+      - /doc
+
+  - name: libcapi20
+    sources:
+      - type: archive
+        url: "http://deb.debian.org/debian/pool/main/libc/libcapi20-3/libcapi20-3_3.27.orig.tar.bz2"
+        md5: 439620a9221c0222c35fb3e7a3973700
+
+  - name: gsm
+    build-options:
+    make-install-args:
+      - INSTALL_ROOT=/app
+      - GSM_INSTALL_INC=/app/include
+    x-native-module:
+      build-options:
+        make-install-args:
+          - GSM_INSTALL_LIB=/app/lib
+    x-compat32-module:
+      build-options:
+        make-args:
+          - CC=ccache i686-unknown-linux-gnu-gcc
+          - CXX=ccache i686-unknown-linux-gnu-g++
+          - LD=i686-unknown-linux-gnu-gcc
+        make-install-args:
+          - CC=ccache i686-unknown-linux-gnu-gcc
+          - CXX=ccache i686-unknown-linux-gnu-g++
+          - LD=i686-unknown-linux-gnu-gcc
+          - GSM_INSTALL_LIB=/app/lib32
+    no-autogen: true
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: "http://www.quut.com/gsm/gsm-1.0.18.tar.gz"
+        sha256: 04f68087c3348bf156b78d59f4d8aff545da7f6e14f33be8f47d33f4efae2a10
+      - type: patch
+        path: ../patches/gsm/makefile.patch
+
+  - name: libusb
+    sources:
+      - type: archive
+        url: "https://github.com//libusb/libusb/releases/download/v1.0.23/libusb-1.0.23.tar.bz2"
+        sha256: db11c06e958a82dac52cf3c65cb4dd2c3f339c8a988665110e0d24d19312ad8d
+    modules:
+
+      - name: eudev
+        cleanup:
+          - /bin
+          - /sbin
+          - /lib*/udev
+        x-native-module:
+          post-install:
+            - udevadm hwdb --update
+        sources:
+          - type: archive
+            url: "https://github.com/gentoo/eudev/archive/v3.2.9.tar.gz"
+            sha256: 7d281276b480da3935d1acb239748c2c9db01a8043aad7e918ce57a223d8cd24
+
+  - name: SPIRV-Headers
+    x-multilib: false
```

Comment:
SPIRV-Headers installs files to lib/cmake/SPIRV-Headers. I think it's safer to enable multilib, so these files will also be installed to lib32.

Reply:
I believe Wine doesn't use cmake and looks up for headers directly.